pkg/document/crdt/root.go

```diff
@@ -151,19 +151,22 @@ func (r *Root) RemovedElementLen() int {
 // GarbageLen returns the count of removed elements.
 func (r *Root) GarbageLen() int {
 	count := 0
+	seen := make(map[string]bool)
 
 	for _, pair := range r.removedElementPairMapByCreatedAt {
-		count++
+		seen[pair.elem.CreatedAt().ToTestString()] = true
```

Comment:
`ToTestString` is intended to return a value for testing purposes, so it does not seem appropriate to use in logic.

```suggestion
		seen[pair.elem.CreatedAt().Key()] = true
```

Reply:
Yes, the changes have been applied.

Actually, I was looking for a function that performs the same role as `toIDString` in `yorkie-js-sdk`, but couldn't find it. Could I know why the names differ between `toIDString` in `yorkie-js-sdk` and `Key` in `yorkie`?